tests/src/Search/Web/DateRangeTest.php

```diff
@@ -16,13 +18,18 @@ public function test_date_range_start_only()
         ]);
 
         $this->newClient();
-        $this->jsonRequest('GET', '/search', ['start-date' => '2016-12-01']);
+        $this->jsonRequest('GET', '/search', ['start-date' => '2016-12-13']);
         $response = $this->getJsonResponse();
-        $this->assertEquals($response->total, 2);
+        $this->assertIds(['15275', '15276'], $response);
 
         $this->jsonRequest('GET', '/search', ['start-date' => '2016-11-01']);
         $response = $this->getJsonResponse();
-        $this->assertEquals($response->total, 3);
+        $this->assertIds(['15275', '15276', '19662'], $response);
+
+        // boundary is included
```

Comment:
The previous assertion could be for `2016-12-05` instead?

Reply:
It's still two different cases: the previous one is a filter that allows all results, while the following is a filter that falls on one of the item dates. Not worth it to go property-based though...

Reply:
Ok; 19662 has a time (ie it's not `00:00:00`), so either use that here or change 15276 to also have a time.

Reply:
The API only allows days, not times, so changing 15276